internal/encrypt/encrypt.go

```diff
@@ -176,6 +176,9 @@ func (e *Encrypter) getImages(count int) ([]string, error) {
 
 	// TODO we can improve this
 	lenImages := len(images)
+	if lenImages == 0 {
+		return nil, errors.New("no image files in /images dir: run 'stego images' to get some random pics")
```

Comment:
Good catch, thanks! Can we use the `dir` variable instead of hardcoding `/images` in the message?

I know that it's hardcoded anyway, but we are going to change it soon I guess. :slightly_smiling_face:

Reply:
Sounds reasonable.